### ISSUE TITLE
[DevResearch] Masthead Accordion - All frameworks

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#2942]
RESEARCH

**Notes:**
not same as masthead menu 

**AC:**
- accessibility audit to be sure masthead Accordion in all frame works meets the criteria of an accessible toggle.

**Comment by guern1kn** [comment]
HTML
- Is not triggered by the Space key
- Needs to be updated to use `<button>`
- We utilize the `aria-controls` in our example, but expect people to put them in. 
- Utilizes `dom-slider` and does not have `display:none` when closed.
1. Focusable and operable with the keyboard
     - [x] 1. In the tab order
     - [x] 2. Triggered by Enter and Space
     - [x] 3. (<Button> will have all of this by default)
 -[x] 2. The trigger includes aria-controls with a value that matches the id of the element that is being controlled. This id is unique on the page. Spark uses lodash to bake this in to our components.
- [x] 3. The trigger includes aria-expanded and it updates correctly when the trigger is activated. This goes on the trigger, not the content.
- [x] 4. When the content is hidden, it is also hidden from screen readers.
    - [x] 1. Must include display:none.
- [x] 5. Not part of the WAI-ARIA best practices but still critical: the trigger element has a label. Either the text of the button, or an aria-label if the text of the button is not appropriate.

React
- Is not triggered by the Space key
- Needs to be updated to use `<button>`
- Need to add `aria-controls`

1. Focusable and operable with the keyboard
   - [x] 1. In the tab order
   - [x] 2. Triggered by Enter and Space
   - [x] 3. (<Button> will have all of this by default)
- [x] 2. The trigger includes aria-controls with a value that matches the id of the element that is being controlled. This id is unique on the page. Spark uses lodash to bake this in to our components.
- [x] 3. The trigger includes aria-expanded and it updates correctly when the trigger is activated. This goes on the trigger, not the content.
- [x] 4. When the content is hidden, it is also hidden from screen readers.
    1. Must include display:none.
- [x] 5. Not part of the WAI-ARIA best practices but still critical: the trigger element has a label. Either the text of the button, or an aria-label if the text of the button is not appropriate.

Angular
1. Focusable and operable with the keyboard
   - [ ] 1. In the tab order
   - [ ] 2. Triggered by Enter and Space
   - [ ] 3. (<Button> will have all of this by default)
- [ ] 2. The trigger includes aria-controls with a value that matches the id of the element that is being controlled. This id is unique on the page. Spark uses lodash to bake this in to our components.
- [ ] 3. The trigger includes aria-expanded and it updates correctly when the trigger is activated. This goes on the trigger, not the content.
- [ ] 4. When the content is hidden, it is also hidden from screen readers.
    1. Must include display:none.
- [ ] 5. Not part of the WAI-ARIA best practices but still critical: the trigger element has a label. Either the text of the button, or an aria-label if the text of the button is not appropriate.

**Comment by guern1kn** [comment]
@ssabo1 We need to create three issues to update each of the Masthead Accordions. One issue per framework. The research is listed above.
